README.md

```diff
@@ -305,42 +305,17 @@ SUBMISSION_BUCKET_NAME=census-eq-dev-1234567-survey-runner-submission ./k8s/depl
 
 ## Internationalisation
 
-We use flask-babel to do internationalisation. To extract messages from source, in the project root run the following command.
+We use flask-babel to do internationalisation. To extract messages from source and create the messages.pot file, in the project root run the following command.
 
 ```
-pipenv run pybabel extract -F babel.cfg -o app/translations/messages.pot .
+make translation-templates
 ```
 
-This will extract messages and place them in the translations/messages.pot file ready for translation.
+This will extract messages and place them in the .pot files ready for translation.
 
-You should only need to create the language files once.
+These .pot files will then need to be translated. The translation process is documented in Confluence [here](https://collaborate2.ons.gov.uk/confluence/display/SDC/Translation+Process)
 
-To create Welsh language files, run the following command
-
-```
-pipenv run pybabel init -i app/translations/messages.pot -d app/translations -l cy
-```
-
-To create the gaelic language files, use the following:
-
-```
-pipenv run pybabel init -i app/translations/messages.pot -d app/translations -l gd
-```
-
-### Compiling the translations
```

Comment:
This section on compiling translations is still valid

Reply:
But isnt this done automatically? because I thought this creates the .mo files and they aren't tracked by git and the second command is covered by ```make translation-templates``` isnt it?

Reply:
Ah yes, this is done as part of the translate command. We should probably document that it exists, but I'll approve this for now to move it on.

Reply:
Cool cheers!